dev-python/matplotlib-inline/matplotlib-inline-0.1.6-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1
+
+DESCRIPTION="Inline Matplotlib backend for Jupyter"
+HOMEPAGE="https://github.com/ipython/matplotlib-inline/"
+SRC_URI="
+	https://github.com/ipython/matplotlib-inline/archive/${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~arm64-macos ~x64-macos"
+
+RDEPEND="
```

Comment:
I suspect it's worth a comment here above RDEPEND for the lack of mpl dep.

Reply:
Is that still necessary considering the optfeatures suggestion?

Reply:
Answering my own question: that was a dumb, sleep-deprived question because we are not talking about the same package.